src/file/document/body/body.ts

```diff
@@ -18,6 +18,9 @@ export class Body extends XmlComponent {
      * @param options new section options
      */
     public addSection(options: SectionPropertiesOptions): void {
+        const currentSection = this.sections.pop() as SectionProperties;
```

Comment:
im having a hard time why the fact of doing a `.pop` will fix issues

Reply:
I’m not sure. I just know that this commit was intended to remove an unnecessary paragraph, which it succeeded at in some cases but in others it has disastrous effects. It makes sense to me to revert it to put the project back into a functioning state. Further optimization can happen after that.

Reply:
Good point, will merge